cwf/gateway/fabfile.py

```diff
@@ -134,7 +134,7 @@ def integ_test(gateway_host=None, test_host=None, trf_host=None,
 
     _switch_to_vm_no_destroy(gateway_host, "cwag_test", "cwag_test.yml")
     execute(_start_ue_simulator)
-    execute(_set_cwag_test_networking, cwag_br_mac)
+    #execute(_set_cwag_test_networking, cwag_br_mac)
```

Comment:
Per what I understand we set up the route on the fab file to be able to easily modify trf server without having to reprovision the box. Not sure if this is being use at all.

Reply:
What would you modify? The IP is hardcoded and cwag has an IP in the same subnet already. Why exclude cwag-test?

Reply:
As mentioned on slack it is better to leave this to the execution time of the test better than on creation of the VM